### DISCUSSION TITLE
Image Resize

### DISCUSSION THREAD

**Discussion by MbakGrace** [teamalgoritma/community#835] [Q&A] [answered]
Dear Team Algoritma,

Mohon ijin bertanya mengenai proses resize image, saya mencoba melakukan resize pada gambar dibawah ini menjadi 600, 400 tapi ketika saya cek kembali untuk shapenya ukurannya menjadi 400, 600 begitu juga sebaliknya, mohon bantuan untuk proses resize dan penjelasan mengenai case ini
![exmpl](https://github.com/teamalgoritma/community/assets/154303680/5b39b5ed-1d91-4f4a-ae00-8be27aa521d1)

**Comment by finesaaa** [comment, marked as answer]
Dear bu @MbakGrace 
Untuk methode `cv2.resize()` itu memiliki format urutan **width** lalu **height** (dalam kasus yang ibu minta adalah width=600 dan height=400
Sedangkan pada output `size.shape` urutan yang dikeluarkan adalah 
[0] axis Y (height)
[1] axis X (width)
[2] axis Z  
Jadi keluarannya itu benar bu 400 -> height, 600 -> width
